src/components/dropdown/dropdown.stories.ts

```diff
@@ -180,6 +180,61 @@ export const itemsAsLinks = (): string => html`
   </calcite-dropdown>
 `;
 
+ItemsAsLinks.storyName = "Items as Links";
+
+export const AMixOfLinksAndNonLinks = (): string => html`
```

Comment:
Let's drop this one since it was removed in the [storybook cleanup PR](https://github.com/Esri/calcite-components/pull/5257/files#diff-bc2137321903a9338f046ec0516d2cd615e61bad7075e894432746133fe121a5).